operators/pkg/instctrl/virtualmachines_test.go

```diff
@@ -287,86 +287,95 @@ var _ = Describe("Generation of the virtual machine and virtual machine instance
 	Context("The environment is persistent", func() {
 		BeforeEach(func() { environment.Persistent = true })
 
-		When("the VM is not yet present", func() {
-			It("Should not return an error", func() { Expect(err).ToNot(HaveOccurred()) })
-
-			It("The VM should be present and have the common attributes", func() {
-				Expect(reconciler.Get(ctx, objectName, &vm)).To(Succeed())
-				Expect(vm.GetLabels()).To(Equal(forge.InstanceObjectLabels(nil, &instance)))
-				Expect(vm.GetOwnerReferences()).To(ContainElement(ownerRef))
-			})
-
-			It("The VM should be present and have the expected specs", func() {
-				Expect(reconciler.Get(ctx, objectName, &vm)).To(Succeed())
-				// Here we overwrite the VM resources, since they would have a different representation due to the
-				// marshaling/unmarshaling process. Still, the correctness of the value is already checked with the
-				// appropriate test case. Additionally, we also overwrite the running value, which is checked in a
-				// different It clause.
-				vm.Spec.Template.Spec.Domain.Resources = forge.VirtualMachineResources(&environment)
-				vm.Spec.Running = nil
-				Expect(vm.Spec).To(Equal(forge.VirtualMachineSpec(&instance, &environment)))
+		contextBody := func(envType clv1alpha2.EnvironmentType) {
```

Comment:
```suggestion
		ContextBody := func(envType clv1alpha2.EnvironmentType) {
```

Just for consistency with the other functions